pages/linux/sar.md

```diff
@@ -0,0 +1,27 @@
+# sar
+
+> Monitor performance of various Linux subsystems.
+
+- Report I/O and Transfer rate every 1 second:
+
+`sar -b 1`
+
+- Report 10 network statistics every 2 seconds for network devices:
+
+`sar -n DEV 2 10`
+
+- Report CPU utilization every 2 seconds:
+
+`sar -u ALL 2`
+
+- Report 20 memory utilization statistics every 1 second:
```

Comment:
This feels a bit awkward. Perhaps we could reword this to something like ...`statistics, one per second`?

Reply:
done..

Reply:
Should it be "once per second" ? I think that sounds better and more natural.

Reply:
On further thought, I think it should be changed to "once every second". Because the other examples are of the form "every 2 seconds", "every 5 seconds" and then you have "one per second", "one per 2 seconds".

Either we use "per second" everywhere or "every second" everywhere. But mixing the 2 does not sound right to me.